daemon/src/common/Defs.h

```diff
@@ -42,6 +42,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 /** Default base package */
 #define DEFAULT_BASE_PAK    PRODUCT_NAME_LOWER
 
+#define DAEMON_FS_LEGACY_PAKS false
```

Comment:
Can we use a cvar instead of a compile-time constant? It might be useful to be able to toggle it without recompilation.